src/network/mcpe/NetworkSession.php

```diff
@@ -1170,6 +1171,10 @@ public function onFormSent(int $id, Form $form) : bool{
 		return $this->sendDataPacket(ModalFormRequestPacket::create($id, json_encode($form, JSON_THROW_ON_ERROR)));
 	}
 
+	public function onFormsClose() : void{
```

Comment:
onFormClose ?

Reply:
players may have more forms queued

Reply:
see `Player::closeForms()` documentation

Reply:
Renamed to `onCloseAllForms` for less confusion